dataland-backend/src/main/kotlin/org/dataland/datalandbackend/health/EdcDummyServerHealthIndicator.kt

```diff
@@ -0,0 +1,11 @@
+package org.dataland.datalandbackend.health
+
+import org.springframework.stereotype.Component
+
+/**
+ * This class customizes the actuator/health endpoint of the backend to display if the EDC-Dummy-Server
+ * is available.
+ */
+@Component
+class EdcDummyServerHealthIndicator :
```

Comment:
wie besprochen - ich halte das für falsch mit den health indicators. Insbesondere wenn "dummy" im namen steht.

Reply:
Siehe meine Antwort auf den Kommentar in der anderen Conversation.

Reply:
ok